proto/ignition/msgs/world_stats.proto

```diff
@@ -61,4 +61,7 @@ message WorldStatistics
 
   /// \brief Iteration step size. It's zero when paused.
   Time step_size = 10;
+
+  /// \brief True is the simulator is stepping, as opposed to running.
```

Comment:
Should we make a note that the `stepping` field can be ignored if the `paused` field is true?

Reply:
432cb99